Library/Formula/delta.rb

```diff
@@ -0,0 +1,52 @@
+require "formula"
+
+# Two requests are needed to download from Tigris.org. In the first, the server
+# provides a cookie and then redirects the client to a servlet. The servlet
+# requires this cookie, or it will send the client in a redirect loop.
+#
+# The Homebrew CurlDownloadStrategy does not store cookies, so this subclass
+# adds the -c flag and a temporary cookie storage file.
+#
+# This relies on internal details of the CurlDownloadStrategy, so expect it to
+# break.
+class CurlWithCookiesDownloadStrategy < CurlDownloadStrategy
+  def cookie_path
+    @cookie_path ||= Pathname.new("#{self.tarball_path}.cookies")
+  end
+
+  def _fetch
+    curl @url, "-c", cookie_path, "-C", self.downloaded_size, "-o",
```

Comment:
actually, only `downloaded_size` is private, but still a good idea to remove the redundant `self` from both